hat/assets/js/apps/Iaso/domains/forms/hooks/useGetPossibleFields.ts

```diff
@@ -66,3 +73,71 @@ export const usePossibleFieldsDropdown = (
         };
     }, [isFetchingForm, possibleFields]);
 };
+
+export type FormVersion = {
+    version_id: string;
+    possible_fields: PossibleField[];
+};
+type FormVersions = {
+    form_versions: FormVersion[];
+};
+type FormVersionResult = {
+    formVersions?: FormVersion[];
+    isFetchingForm: boolean;
+};
```

Comment:
I find those names confusing. 
```suggestion
export type FormVersionPossibleFields = {
    version_id: string;
    possible_fields: PossibleField[];
};
type FormVersionsPossibleFields = {
    form_versions: FormVersion[];
};
type FormVersionResult = {
    formVersionsPossibleFields?: FormVersionPossibleFields[];
    isFetching: boolean;
};
```